MediaAdminBundle/Resources/public/coffee/extendView/orchestraAbstractMediaType.coffee

```diff
@@ -10,14 +8,10 @@ extendView['orchestraMediaType'] =
     previewId = '#previewImage_' + target.data('input')
     $(inputId).val ''
     $(previewId).removeAttr 'src'
-  openMediaModal: (event) ->
-    event.preventDefault()
-    @method = if @options.method then @options.method else 'GET'
-    target = $(event.currentTarget)
-    modal = $('#' + target.data("target"), @$el)
-    inputId = target.data("input")
-    url = target.data("url")
-    viewClass = appConfigurationView.getConfiguration('media', 'showMediaModal')
+
+  openMediaModal: (modal, inputId, url, method, mediaModalView) ->
+    mediaModalView = mediaModalView or "showMediaModal"
```

Comment:
Can you use instead the feature of coffee allowing to declare default value in function signature ?
openMediaModal: (modal, inputId, url, method, mediaModalView = "showMediaModal")